### PR TITLE
#494 filter hooks documentation added @ possible places (#494)

### PR DESCRIPTION
**Pull request by amolv**

### CODE REVIEW

File: inc/template-tags.php [hunk 1]
```diff
@@ -192,7 +203,20 @@ function twentytwenty_get_post_meta( $post_id = null, $location = 'single-top' )
 
 	// Get the post meta settings for the location specified.
 	if ( 'single-top' === $location ) {
-
+		/**
+		 * Filters post meta info visibility
+		 *
+		 * Use this filter to hide post meta information like Author, Post date, Comments, Is stikcy status
```
Comment (carolinan) [inline inc/template-tags.php]: There is a tiny typo here:   _Is stikcy status_

Comment (amolv) [reply inc/template-tags.php]: Typo fixed.

**Comment by amolv** [comment]
> Please update the files to solve the problems found in the Travis test.

Waiting for Travis build

**Comment by pattonwebz** [comment]
Hey @amolv if you want to enable the ability for me to push into this branch I can show you where the changes need made and fix these. Instructions for enabling maintainers push can be read here: https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by amolv** [comment]
> There are some whitespace/phpcs issues shown in travis. Any chance we could resolve those?

Resolved!

**Pull request merged**